### ISSUE TITLE
Package translations not working with Meteor 1.3 (beta 8)

### ISSUE THREAD

**Issue by nicooprat** [TAPevents/tap-i18n#143]
Couldn't make it work with latest version of Meteor 1.3. Tried a lot of configurations based on the docs and code from live apps like Telescope, but all strings was displayed with their keys and not their values. No problem with project-level translations though.

Anyone got it to work ? Is it a known issue ?

**Comment by SachaG** [comment]
I think it's a known issue. For [Nova](https://github.com/TelescopeJS/Telescope/tree/nova/) I've temporarily removed i18n.

**Comment by Andrew1431** [comment]
Any progress on this...

**Comment by luckyyang** [comment]
Any progress on this？

**Comment by Andrew1431** [comment]
@luckyyang we moved our translations from each package in to a seperate translations package that loads before all other packages. This solution got all the translations working properly.

**Comment by nicooprat** [comment]
I stopped using packages at all and followed more or less the Mantra spec https://github.com/kadirahq/mantra

**Comment by derwaldgeist** [comment]
Any solution for this?

**Comment by deehuey** [comment]
Yeah, with latest Meteor you can use npm modules, so I would suggest just switching to the normal npm tapi18n package, and using imports.

**Comment by derwaldgeist** [comment]
Hm. How exactly does that solve my problem that i18n does not work in Meteor packages? I don't want to switch the app's packages to npm packages, because I want to keep it in a modular fashion. Reason is that I am sharing a lot of the Meteor packages across apps.
